static/js/components/email/EmailCompositionDialog.js

```diff
@@ -86,6 +87,42 @@ export default class EmailCompositionDialog extends React.Component {
     }
   }
 
+  insertRecipientVariable = (variableName: string) => {
+
+    const { editorState } = this.state;
+    if(editorState !== null && editorState !== undefined) {
```

Comment:
Probably you should update the type for `this.state` then, from `editorState: ?Object` to `editorState: Object`. `?Object` is no longer correct, since we can't really get into a state where it is `null` or `undefined`